src/apps.config.js

```diff
@@ -27,6 +28,15 @@ const apps = [
         desktop_shortcut: false,
         screen: displayTodoist,
     },
+    {
+        id: "calc",
+        title: "Calc",
```

Comment:
change new app's name from 'calc' to 'calculator'

Reply:
hey! , in ubuntu, the name is "Calc" so should I change it or not?

Reply:
well then it's fine, leave it

Reply:
Thanks!